setup/react/app/ApplicationBootstrap.js

```diff
@@ -22,7 +22,7 @@ const ApplicationBootstrap = ({ children }) => {
   useTransactionUpdate();
```

Comment:
Not related to this file:
Adding Enevti then trying to remove it crashes the application 
<img width="871" alt="image" src="https://user-images.githubusercontent.com/8784876/227495639-391719d1-00cd-4615-9d23-deb4e0f76148.png">

Reply:
I guess I would raise this in another issue because trying to apply fixes here would be out of scope of this ticket @oskarleonard

Reply:
I added it to my network/application issue